signalfx/splunk-otel-collector-chart
# Adding pprof extension

Adding Prof extension

Testing:

- Check with nordport and headless. 
- Check with enabled and disabled pprof.
- Check with multiple nodes.
